src/widgets/HazardPoint.js

```diff
@@ -20,14 +20,22 @@ const hazardPath = `M569.517 440.013C587.975 472.007 564.806 512 527.94 512H48.0
  * Custom component for rendering clickable hazard icons.
  */
 export const HazardPoint = props => {
-  const { x, y, onPress, datum } = props;
+  const { x, y, onPress, dataSet, index } = props;
   const { xOffset, yOffset, scale, fill } = hazardPointStyles;
 
-  const onPressWrapper = React.useCallback(() => onPress?.(datum?.id), []);
+  const onPressWrapper = React.useCallback(() => onPress?.(dataSet[index]?.id), []);
 
   return (
     <Svg>
-      <G onPressIn={onPressWrapper} delayPressIn={0}>
+      <G>
+        <Rect
```

Comment:
Could you swap the order of the `Rect` and the `Path` please?
On my emulator I have to click just beside the hazard icon to get the modal to show, pressing on the hazard icon doesn't do anything. Changing the order here puts the rectangle on top and enables the press on the icon.

Reply:
Oh nice! 3b2301e